UT4MasterServer.Authentication/UT4MasterServer.Authentication.csproj

```diff
@@ -7,8 +7,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.Authentication" Version="2.2.0" />
-    <PackageReference Include="Microsoft.AspNetCore.Authorization" Version="6.0.13" />
+    <PackageReference Include="Microsoft.AspNetCore.Authorization" Version="8.0.4" />
+    <PackageReference Include="Microsoft.AspNetCore.Identity.UI" Version="6.0.29" />
```

Comment:
Why are you mixing Microsoft.AspNetCore.* version dependencies?

Reply:
Because I havent worked with .NET in a while and NuGet is telling me to update packages to 8.*.*. Newertheless I'll change it to highest 6.*.* version.